templates/base.html

```diff
@@ -128,4 +128,4 @@ <h4 id="inspiration">Still need inspiration?</h4>
       ga('send', 'pageview');
     </script>
   </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
Ack, the Web-based editor in GitHub automatically added this line break.
(Not that I mind, but it isn't related to the change I'm actually
proposing.)